lib/srv/reexec.go

```diff
@@ -539,6 +540,7 @@ func RunNetworking() (errw io.Writer, code int, err error) {
 	// Use stderr so that it's not forwarded to the remote client.
 	errorWriter := os.Stderr
 
+	log := slog.New(slog.NewJSONHandler(errorWriter, nil))
```

Comment:
I'm not sure 🤔 I actually never saw these logs in the agent logs or in the client terminal, so it might be safer to omit this entirely. We already send a message to the client's terminal when we have to fallback to `/`